doc/adr/0008-concept-for-handling-ldml.de-in-backend.md

```diff
@@ -0,0 +1,53 @@
+# 6. Concept for handling LDML.de in the backend
+
+Date: 2024-04-12
+
+## Status
+
+Proposed
+
+## Context
+
+We have identified two main issues with our current approach in the backend source code:
```

Comment:
If nothing changes with the bullet points below, it should be "three" instead of "two.